src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +339,73 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    static String[] getDockerGroupCmdArgs(String scanReportPath, Boolean scanWithNotRootUser) {
+        String[] cmdGroupArgs = {"", ""};
+        if(!scanWithNotRootUser)
```

Comment:
We should include curly braces IMO.

Reply:
src/main/java/com/neuvector/Scanner.java